src/sentry/search/snuba/backend.py

```diff
@@ -49,6 +50,19 @@ def unassigned_filter(unassigned, projects):
     return query
 
 
+def triaged_filter(triaged, projects):
+    query = Q(
+        id__in=GroupLink.objects.filter(
+            project_id__in=[p.id for p in projects],
+            linked_type=GroupLink.LinkedType.issue,
+            relationship=GroupLink.Relationship.references,
+        ).values_list("group_id", flat=True)
+    )
+    if not triaged:
+        query = ~query
+    return query
```

Comment:
This looks good to me, but will miss groups linked via our integration platform (https://docs.sentry.io/product/integrations/integration-platform/). I think we'd need an OR here on the  PlatformExternalIssue model.